Sources/Utility/Git.swift

```diff
@@ -40,21 +40,10 @@ public class Git {
             }
         }
 
-        // Otherwise, look for normal tags.
+        // Otherwise, look for normal and 'v'-prefixed tags.
         for tag in tags {
-            if let version = Version(string: tag) {
-                knownVersions[version] = tag
-            }
-        }
-
-        // If we didn't find any versions, look for 'v'-prefixed ones.
-        //
-        // FIXME: We should match both styles simultaneously.
-        if knownVersions.isEmpty {
-            for tag in tags {
-                if let version = Version.vprefix(tag) {
-                    knownVersions[version] = tag
-                }
+            if let version = Version(string: tag) ?? Version.vprefix(tag) {
```

Comment:
Both for loops could be merged now:

```swift
for tag in tags {
    var trimmedTag = tag
    for versionSpecificKey in Versioning.currentVersionSpecificKeys {
        if tag.hasSuffix(versionSpecificKey) {
            trimmedTag = String(tag.dropLast(versionSpecificKey.count))
            break
        }
    }

    if let version = Version(tag: trimmedTag) {
        knownVersions[version, default: []].append(tag)
    }
}
```

Reply:
Much nicer! I'll make these changes ASAP